scene/gui/line_edit.h

```diff
@@ -259,26 +259,26 @@ class LineEdit : public Control {
 	void set_placeholder_alpha(float p_alpha);
 	float get_placeholder_alpha() const;
 
-	void set_cursor_position(int p_pos);
-	int get_cursor_position() const;
+	void set_caret_position(int p_pos);
+	int get_caret_position() const;
```

Comment:
Wonder if it's worth renaming these to `[set|get]_caret_column` to match `TextEdit`?

Reply:
Updated the property name to `caret_column` along with the getters and setters.